src/main/java/yow/Ui.java

```diff
@@ -0,0 +1,37 @@
+package yow;
+/**
+ * Handles user interactions, including displaying messages and prompting for input.
+ */
+public class Ui {
+
+    private void splitLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    void prettyPrint(String text) {
+        splitLine();
+        System.out.println(text);
+        splitLine();
+    }
+
+    void helpChat() {
```

Comment:
Perhaps helpChat() could be a verb instead?
```suggestion
    void showHelpChat() {
```